### PR TITLE
Suite2p

### PR DESCRIPTION
**Pull request by Saksham20**
this uses a converter like the one from nwb-conversion-tools

### CODE REVIEW

**Comment by Saksham20** [comment]
@bendichter  to save suite2p to nwb: 
```
from segmentationextractors.suite2p.suite2psegmentationextractor import Suite2pSegmentationExtractor
from segmentationextractors.nwbextractor.nwbsegmentationextractor import NwbSegmentationExtractor

fileloc = 'path-to-DropBoxfile' # please change the default folder name to 'combined', since suite2p looks for this folder
suite2p_extractor = Suite2pSegmentationExtractor(fileloc)

metafile = r'SegmentationExtractors/tests/metadatafiles/nct_ophys_metafile_suite2p.yaml' 
flname=r'saveloc\*.nwb'

NwbSegmentationExtractor.write_recording(suite2p_extractor, flname, metadata_dict=metafile)
```
instead of inputing various names into the nwbwrite function, it takes in a yaml file instead, which has prefilled values that can be over ridded. 
With this method of entering metadata, the gui can also be leveraged to create the file.

File: segmentationextractors/schnitzerextractor/cnmfesegmentationextractor.py [hunk 1]
```diff
@@ -22,31 +23,23 @@ def __init__(self, filepath):
         self._dataset_file, self._group0 = self._file_extractor_read()
         self.extimage_dims, self.raw_images = self._image_mask_extractor_read()
         self.image_masks = self.raw_images
-        self.roi_response = self._trace_extractor_read()
+        self.roi_response = self._trace_extractor_read('extractedSignals')
+        self.roi_response2 = self._trace_extractor_read('extractedPeaks')
```
Comment (bendichter) [inline segmentationextractors/schnitzerextractor/cnmfesegmentationextractor.py:27]: this seems like a bad name for this

File: segmentationextractors/schnitzerextractor/extractsegmentationextractor.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import numpy as np
 import h5py
-from ..segmentationextractor import SegmentationExtractor
+from segmentationextractors.segmentationextractor import SegmentationExtractor
```
Comment (bendichter) [inline segmentationextractors/schnitzerextractor/extractsegmentationextractor.py:3]: please change back to relative import

**Comment by bendichter** [comment]
`NwbSegmentationExtractor.write_recording` should not take in a yaml, but a dict of dicts read from the yaml

File: tests/metadatafiles/nct_ophys_metafile_suite2p.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  sex: F
+  species: species
+  weight: weight number
+  date_of_birth: 2010-02-02 00:00:00
+Ophys:
+  Device:
+  - name: Microscope
+    description: ophys_device1_description
+  DfOverF:
+    name: df/f_name
```
Comment (bendichter) [inline tests/metadatafiles/nct_ophys_metafile_suite2p.yaml:31]: names should not have slashes in them

**Comment by bendichter** [comment]
The main `RoiResponseSeries` that is currently named "Fluorescence" should be named "RoiResponseSeries"

All of the `RoiResponseSeries` objects have the wrong data orientation. Time is in the 0th axis for all `TimeSeries` objects.

**Pull request closed without merging**